components/Card.js

```diff
@@ -21,56 +42,91 @@ function Card({ backgroundImageSource, backgroundImageAltText, children }) {
         </figure>
       )}
       <div
-        className={`card-body grid ${cardsBackgroundColor} text-${textColor} gap-y-3 auto-rows-card`}
+        className={`carousel-body flex flex-col p-6 shadow-lg justify-start items-center ${cardsBackgroundColor} text-${textColor} rounded-b-md`}
       >
-        {children}
+        <CardTitle>{title}</CardTitle>
+        {(isHover || simulateHover) && (
+          <hr
+            className={
+              'h-1  bg-teal-medium border-none text-teal-medium duration-200 w-full my-1'
+            }
+          />
+        )}
+        {paragraphs?.map((paragraph, index) => (
+          <CardParagraph key={index}>{paragraph}</CardParagraph>
+        ))}
+        {/* not a good practice to use index as key will change later */}
+        {address && <CardAddress>{address}</CardAddress>}
+        {listLabel && (
+          <PositionRequirements items={listItems} label={listLabel} />
+        )}
+        {(isHover || simulateHover) && action && (
+          <CardAction
+            actionCost={actionCost}
+            cause={cause}
+            isExternal={isExternal}
+            link={link}
+          >
+            {action} {isAvailable(actionCost) && `$${actionCost.toString()}`}
+          </CardAction>
+        )}
       </div>
       <style jsx>{`
         .card-shadow {
           box-shadow: rgba(14, 30, 37, 0.061) 6px 6px 12px 0px,
             rgba(14, 30, 37, 0.075) 6px 6px 10px 0px;
         }
+        .carousel-card {
+          min-width: 350px;
+          max-width: 350px;
+        }
+        .carousel-body {
+          min-height: 420px;
```

Comment:
I think this is a good number as per the design file, but the problem is that it's not including the image size. The overall card (carousel-body) should be this size, but this is the image size + card (carousel-body). I recommend changing the class name to card-body and making it account for the whole card body including the image.

Reply:
sure!

Reply:
Just to keep a note of our conversation. We'll change the content that seems to be creating overflow issues here after the PR is pushed through. Let's just get to a place where we don't have the cards jumping in size 🦖